doc/source/guide/geometry_guide.rst

```diff
@@ -0,0 +1,286 @@
+
+In three dimensions, there is no unique way to rotate one vector to another, which is why a convention is required in this case.
+The standard configuration in 3d is
+
+    .. math::
+        r(0) =
+        \begin{pmatrix}
+            0 \\
+            1  \\
+            0
+        \end{pmatrix},
+        \
+        s(1, 0) =
+        \begin{pmatrix}
+            1 \\
+            0  \\
+            0
+        \end{pmatrix},
+        \
+        s(0, 1) =
+        \begin{pmatrix}
+            0 \\
+            0  \\
+            1
+        \end{pmatrix}
+
+for initial detector position and axes.
+Here the zero parameter in :math:`r(0)` can have one or more components, and if the detector is only one-dimensional, we use :math:`s(0) = (1, 0, 0)^{\mathrm{T}}` only.
+
+.. figure:: figures/coord_sys_3d.svg
+    :width: 50%
+    :align: center
+
+    Default 3D coordinate system.
+    The configuration is chosen such that the detector axes coincide with the standard :math:`x` and :math:`z` coordinate axes.
+
+The transition to other initial states is done by rotating the whole system, and the rotation is either explicitly specified or computed to rotate one vector to another.
+Let :math:`v_1, ..., v_K \in \mathbb{R}^3` be the vectors defining the initial configuration of a 3d geometry, and :math:`v_1^{(\text{def})}, ..., v_K^{(\text{def})}` are their default values.
+The rotated configuration is given by
+
+    .. math::
+        v_k = R^{(\mathrm{i})}\, v_k^{(\text{def})},
+
+i.e., *all vectors are transformed by the same rotation*.
+The matrix :math:`R^{(\mathrm{i})} \in \mathbb{R}^{3 \times 3}` is chosen to rotate the first vector from its default :math:`v_1^{(\text{def})}` to its actual value :math:`v_1`, i.e.,
+
+    .. math::
+        R^{(\mathrm{i})} v_1^{(\text{def})} = v_1.
+
+Since the rotation :math:`R^{(\mathrm{i})}` is not uniquely determined, we choose to perform a rotation in the plane spanned by :math:`v_1^{(\text{def})}` and :math:`v_1`, making use of `Rodrigues' rotation formula <https://en.wikipedia.org/wiki/Rodrigues%27_rotation_formula>`_:
+
+    .. math::
+        a &= \frac{v_1^{(\text{def})} \times v_1}{\big| v_1^{(\text{def})} \times v_1 \big|},\ \cos\beta = \langle v_1^{(\text{def})},\, v_1 \rangle,
+
+        R^{(\mathrm{i})} v &= \cos\beta v + \sin\beta (a \times v) + (1 - \cos\beta)\langle a,\, v \rangle a.
+
+This construction becomes unstable when :math:`v_1^{(\text{def})} \approx v_1`.
+Therefore, if :math:`\big\| v_1^{(\text{def})} - v_1 \big\| < \delta` for a threshold :math:`\delta`, we take :math:`v_1 = v_1^{(\text{def})}`.
+
+Below are two concrete examples for how this computation is done in practice.
+For a list of implemented geometries, check the API documentation of the ``odl.tomo.geometry`` subpackage.
+
+`Parallel3dEulerGeometry`
```

Comment:
Now we have, and it required some fixes in the backend. The adjoint is off by a large factor, and I don't really know why. Needs more investigation, but I suggest that we don't hold up this PR for that.